bpf/lib/conntrack.h

```diff
@@ -606,30 +633,7 @@ ct_extract_ports4(struct __ctx_buff *ctx, int off, enum ct_dir dir,
 	return 0;
 }
 
-/* The function determines whether an egress flow identified by the given
- * tuple is a reply.
- *
- * The datapath creates a CT entry in a reverse order. E.g., if a pod sends a
- * request to outside, the CT entry stored in the BPF map will be TUPLE_F_IN:
- * pod => outside. So, we can leverage this fact to determine whether the given
- * flow is a reply.
- */
-static __always_inline int
-ct_is_reply4(const void *map, struct __ctx_buff *ctx, int off,
-	     struct ipv4_ct_tuple *tuple, bool *is_reply)
-{
-	int err = 0;
-
-	err = ct_extract_ports4(ctx, off, CT_EGRESS, tuple);
-	if (err < 0)
-		return err;
-
-	tuple->flags = TUPLE_F_IN;
-
-	*is_reply = map_lookup_elem(map, tuple) != NULL;
-
-	return 0;
-}
+CT_IS_REPLY(4)
```

Comment:
Maybe add a comment `/* This defines ct_is_reply() function */`, so it can be grep-able?